Update network-watcher-read-nsg-flow-logs.md

mattreatMSFT opened the pull request:
Changed initial and final block size to reflect version currently in production. Added MAC Address to NSG blob path

azurecla commented:
Hi __@woutercx__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Thank you for this contribution - sending to the author for review.

cynthn commented:
@woutercx this is for attaching an existing data disk?

woutercx commented:
Hi,

I don't remember exactly what I did, but yes, I think it was an existing 
data disk.
I took an image from the VM Depot Azure Gallery:
http://www.hanselman.com/blog/Over400VirtualMachineImagesOfOpenSourceSoftwareStacksInTheVMDepotAzureGallery.aspx
and in the end I had to make it writable.

On 06/11/15 19:37, Cynthia Nottingham [MSFT] wrote:
>
> @woutercx <https://github.com/woutercx> this is for attaching an 
> existing data disk?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Azure/azure-content/pull/4822#issuecomment-154495621>.
>

dsk-2015 commented:
@woutercx thanks for your contribution. I just tried this with the new disk, and you do need to make it writable with that extra command. Please go ahead and merge this change, after making these changes: 
1) Replace datadisk by datadrive as that is used in the article 
2) Fix the typo in the second step, datadrive/ should be replaced with /datadrive 

Thanks again for doing this.

tysonn commented:
@woutercx - Dhanashri submitted some patches to your fork for this - if you could accept her pull request in your fork of this project, we'll be able to move ahead and accept your update :)

Here's the link to that pull request: https://github.com/woutercx/azure-content/pull/1

tysonn commented:
Thanks!

mattreatMSFT committed:
Update network-watcher-read-nsg-flow-logs.md

Changed initial and final block size to reflect version currently in production. Added MAC Address to NSG blob path
